src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -432,9 +432,12 @@ public void close() throws Exception {
     isClosed = true;
     this.channelCache.closeAllChannels();
 
-    // Collect the perf metrics before closing if needed
-    if (metrics != null) {
-      Slf4jReporter.forRegistry(metrics).outputTo(logger.getLogger()).build().report();
+    // unregister jmx metrics
+    if (this.metrics != null) {
+      removeMetricsFromRegistry();
+
+      // LOG jvm and thread metrics at the end
```

Comment:
There're a few other places in this PR also have this problem
```suggestion
      // LOG jvm memory and thread metrics at the end
```

Reply:
will change